keps/sig-network/563-dual-stack/README.md

```diff
@@ -971,10 +973,13 @@ CoreDNS will need to make changes in order to support the plural form of
 endpoint addresses. Some other considerations of CoreDNS support for
 dual-stack:
 
-- Because service IPs will remain single-family, pods will continue to access
-  the CoreDNS server via a single service IP. In other words, the nameserver
-  entries in a pod's /etc/resolv.conf will typically be a single IPv4 or single
-  IPv6 address, depending upon the IP family of the cluster's service CIDR.
+- For dual-stack services, CoreDNS will be updated to read both IPFamilies
+  assigned to the service.
+- For single-stack services, CoreDNS will operate as is. Service IPs will remain 
+  single-family, pods will continue to access the CoreDNS server via a single service 
+  IP. In other words, the nameserver entries in a pod's /etc/resolv.conf will typically
+  be a single IPv4 or single IPv6 address, depending upon the IP family of the cluster's 
+  service CIDR.
```

Comment:
is this needed?  if your DNS nameserver is fd00:1:2:3::10 you can only use IPv6 to access it, same if it is 192.168.1.1, you can use IPv4 only.
If the resolv.conf doesn't have the DNS serviceIP is a bug

Reply:
I am not sure i understand the issue here.

Reply:
I was thinking if this is somehow redundant, just wondering why do you want to make it explicit, not a big deal, disregard the comment if you want.

Reply:
Thanks.